jiant/utils/utils.py

```diff
@@ -343,7 +343,7 @@ def load_model_state(model, state_path, gpu_id, skip_task_models=[], strict=True
         not (skip_task_models and strict),
         "Can't skip task models while also strictly loading task models. Something is wrong.",
     )
-
+    strict = False
```

Comment:
I'm not familiar with this setting. Can you tell me more about why we're hard-coding this here?

Reply:
Oh whoops this was for debugging one of the performance runs. I'll remove this